### PR TITLE
Story/quad 88 sprint 2

### PR DESCRIPTION
**Pull request by sarvani11**
updated navbar.jsp all the tags are aligned

### CODE REVIEW

File: Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp [hunk 1]
```diff
@@ -1,4 +1,6 @@
 <li ${currentPage == "home" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}">Home</a></li>
 <li ${currentPage == "networks" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/browsenetworks">Networks</a></li>
 <li ${currentPage == "exploregraph" ? "class=\"active\"" : ""}><a href="">Explore</a></li>
-<li ${currentPage == "networksearch" ? "class=\"active\"" : ""}><a href="">Search</a></li>
\ No newline at end of file
+<li ${currentPage == "networksearch" ? "class=\"active\"" : ""}><a href="">Search</a></li>
+<li><a class="fa fa-user" href="/quadriga/j_spring_security_logout" style=" margin:2px 10px; padding:0px 0px 0px 0px; font-size:80% ">${project.owner.name} <br> &nbsp Logout</a>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp]: the logout link should only be on "Logout" not on the username as well.

**Comment by jdamerow** [comment]
@sarvani11 please take a look at this.

**Comment by sarvani11** [comment]
@jdamerow  resolved all the conflicts and the story has been done. Please review and let me know if any changes need to be made.All the above 5 commits have been made by me from @MeghanTalasila  laptop as my Quadriga environment is not functioning.

File: Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp [hunk 2]
```diff
@@ -1,3 +1,12 @@
+<<<<<<< HEAD
+<li ${currentPage == "home" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}">Home</a></li>
+<li ${currentPage == "networks" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/browsenetworks">Networks</a></li>
+<li ${currentPage == "exploregraph" ? "class=\"active\"" : ""}><a href="">Explore</a></li>
+<li ${currentPage == "networksearch" ? "class=\"active\"" : ""}><a href="">Search</a></li>
+<li> <a class="fa fa-user" style=" margin:2px 10px; padding:0px 0px 0px 0px; font-size:80%">${project.owner.name} </a>
+<a class="fa fa-user" href="/quadriga/j_spring_security_logout" style=" margin:2px 10px; padding:0px 0px 0px 0px; font-size:80% "> Logout </a></li>
+
+=======
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp:9]: conflict

**Comment by sarvani11** [comment]
cleared the conflicts, merged successfully. Please review and let me know if any changes are required

**Pull request merged**